### ISSUE TITLE
OLED display footprint

### ISSUE THREAD

**Issue by sslupsky** [TG9541/kicad-footprints#2]
Thanks for the footprint.  Quick question regarding the "new" versus "old", is the only difference the dimension?  I think I noticed an extra capacitor on the "old" version.  Somewhere I recall reading something about some displays have a voltage regulator and some others don't.

I recall coming across one version where the power and ground pins were reversed as well.

**Comment by sslupsky** [comment]
Also noticed that the left vertical silkscreen line is slightly skewed
![Screen Shot 2022-03-23 at 3 55 05 PM](https://user-images.githubusercontent.com/16888734/159802529-d07320b7-1003-4379-8994-f5376bdd56e1.png)

**Comment by TG9541** [comment]
You're right, the silkscreen is not perfect... The only difference is the shape of the drilled holes. 

I only had two samples: "old" (wider) and "new" (narrower). There may - very well - be more variant out there.

Please go ahead and fork the repo to provide an improved variant - maybe even one that adds jumper pads for different pin assignments.
